source/MRMesh/MRObjectVoxels.cpp

```diff
@@ -232,6 +239,12 @@ void ObjectVoxels::setActiveBounds( const Box3i& activeBox, ProgressCallback cb,
         accessor.setActiveState( {x,y,z}, insideX && insideY && insideZ );
         reportProgress( cb, [&]{ return cbModifier * float( counter ) / volume; }, ++counter, 256 );
     }
+
+    volumeRenderActiveVoxels_.clear();
+    auto newDims = activeBox_.size();
+    volumeRenderActiveVoxels_.resize( newDims.x * newDims.y * newDims.z, true );
+    dirty_ |= DIRTY_SELECTION;
```

Comment:
why make it here? better in `prepareDataForVolumeRendering`

Reply:
we change active box, it can change box size, so if we try to get active voxels, we will get wrong data